include/boost/test/impl/framework.ipp

```diff
@@ -270,7 +270,7 @@ private:
 
         // look for match
         // using namespace boost::placeholders;
-        return std::find_if( filters.begin(), filters.end(), bind( &component::pass, _1, boost::ref(tu) ) ) != filters.end();
+        return std::find_if( filters.begin(), filters.end(), bind( &component::pass, placeholders::_1, boost::ref(tu) ) ) != filters.end();
```

Comment:
Is this a change from what was on develop? We need to mark any changes we make in third party code with `// airtime: begin` comments, but it would be good to understand what's going on here.

@artasano @IgorKolo Any ideas?

Reply:
Yes, using the merged changes in `airtime-master` caused a namespace error here when it was ok with merged `develop`. 

Oh right, I'll need to add that comment.

Reply:
Oh, I see. On boostorg's master branch that line 272 is uncommented. This file had a merge conflict when merging with `airtime-master` and we must have had it commented out or something and I took our revision. I can revert this change and have it match boostorg.